build.gradle

```diff
@@ -931,6 +936,20 @@ task runQuickstart {
       if (conffile.exists())
         conffile.delete()
     }
+    //Setting _JAVA_OPTIONS to avoid using gradle settings which is 256m which was causing rdd to
+    // overflow on disk
+    Map<String,Object> env = new HashMap<String,Object>()
+    env.put("_JAVA_OPTIONS","-Xms256m -Xmx4g")
```

Comment:
When a process from gradle (using exec) was spawned it uses XmX and Xms values of gradle even though they are set explicitly but setting _JAVA_OPTIONS worked.So I have done this way.This is only for Quickstart.scala script